### PR TITLE
Updated NSLayoutConstraint methods in source files

### LINKED ISSUE #17: Soon to be deprecated NSLayoutConstraint methods
Are there plans to move the use of -addConstraint, etc. over to the suggested methods? I'd be glad to submit the PR.

See:
screen shot 2016-02-13 at 11 38 27 pm

### PR DESCRIPTION
**Pull request by gonzalonunez**
Also fixes #17. Sorry about the mix up!

### CODE REVIEW

File: Source/AnimatedPagingScrollViewController.swift [hunk 1]
```diff
@@ -46,14 +46,16 @@ public class AnimatedPagingScrollViewController : UIViewController, UIScrollView
...
+        NSLayoutConstraint.activateConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:|[scrollView]|", options: [], metrics: nil, views: ["scrollView" : scrollView]))
+        NSLayoutConstraint.activateConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[scrollView]|", options: [], metrics: nil, views: ["scrollView" : scrollView]))
+        let scrollViewLeft = NSLayoutConstraint(item: scrollView, attribute: .Left, relatedBy: .Equal, toItem: contentView, attribute: .Left, multiplier: 1, constant: 0)
+        let scrollViewRight = NSLayoutConstraint(item: scrollView, attribute: .Right, relatedBy: .Equal, toItem: contentView, attribute: .Right, multiplier: 1, constant: 0)
+        let scrollViewTop = NSLayoutConstraint(item: scrollView, attribute: .Top, relatedBy: .Equal, toItem: contentView, attribute: .Top, multiplier: 1, constant: 0)
+        let scrollViewBottom = NSLayoutConstraint(item: scrollView, attribute: .Bottom, relatedBy: .Equal, toItem: contentView, attribute: .Bottom, multiplier: 1, constant: 0)
+        NSLayoutConstraint.activateConstraints([scrollViewLeft, scrollViewRight, scrollViewTop, scrollViewBottom])
```
Comment (maxmeyers) [inline Source/AnimatedPagingScrollViewController.swift:58]: Can you add a line of whitespace between each group of variables and `activateConstraints` calls?

**Comment by gonzalonunez** [comment]
Fixed whitespace and squashed the commits. Let me know if there is anything else that needs to be done! :+1:

**Comment by maxmeyers** [comment]
I'll update and push the podspec soon.

**Comment by gonzalonunez** [comment]
Great!

**Comment by maxmeyers** [comment]
Updated :+1:

**Pull request merged**